### PR TITLE
PR for new security article, artwork, and new TOC node

### PR DESCRIPTION
**Pull request by LouisBerner**

### CODE REVIEW

**Comment by Jak-MS** [comment]
@alkohli @dansisson 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.

NOTE: because this PR contains a new article and content, the commits will need to be moved to the private repo. Let us know if you'd like assistance. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-stack/hci/concepts/other-security-features.md [hunk 1]
```diff
@@ -0,0 +1,518 @@
...
+> We recommend that you create your own local administrator account, and also that you disable the well-known `RID 500` user account.
+
+The `Guest` account is protected by the security baseline drift control mechanism. For more information, see [Security baseline settings for Azure Stack HCI (preview)](/azure-stack/hci/concepts/secure-baseline).
+
+We also added two net new components to Azure Stack HCI in 23H2 release, as follows:
+
+- Secrets management engine.
+- SIEM Syslog forwarder agent.
+
+## Secrets management engine
```
Comment (ManikaDhiman) [inline azure-stack/hci/concepts/other-security-features.md:37]: Consider rephrasing the heading text to: 'Manage secrets in Azure Stack HCI'. 

This edit requires SME validation.

**Comment by ManikaDhiman** [comment]
@LouisBerner, @alkohli: I've provided my review comments. However, I am still unconvinced with the article title and intro content.

File: azure-stack/hci/concepts/other-security-features.md [hunk 2]
```diff
@@ -0,0 +1,518 @@
...
+
+- **Syslog over TCP with mutual authentication (client and server) and TLS 1.2 encryption:** In this configuration, both the syslog server and the syslog client can verify the identity of each other via certificates. Messages are sent over a TLS 1.2 encrypted channel.
+- **Syslog over TCP with server authentication and TLS 1.2 encryption:** In this configuration, the syslog client can verify the identity of the syslog server via a certificate. Messages are sent over a TLS 1.2 encrypted channel.
+- **Syslog over TCP with no encryption:** In this configuration, the syslog client and syslog server identities aren’t verified. Messages are sent in clear text over TCP.
+- **Syslog over UDP with no encryption:** In this configuration, the syslog client and syslog server identities aren’t verified. Messages are sent in clear text over UDP.
+
+  >[!IMPORTANT]
+  > To protect against man-in-the-middle attacks and eavesdropping of messages, Microsoft strongly recommends that you use TCP with authentication and encryption in production environments.
+
+## Configuring SYSLOG Security Events forwarder
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:113]: Content revised. Thx.

File: azure-stack/hci/concepts/other-security-features.md [hunk 3]
```diff
@@ -0,0 +1,518 @@
...
+
+### PowerShell cmdlet reference
+
+Use the following cmdlet to pass the syslog server information to the forwarder and to configure the transport protocol, the encryption, the authentication, and the optional certificate used between the client and the server.
+
+```azurepowershell
+Set-AzSSyslogForwarder [-ServerName <String>] [-ServerPort <UInt16>] [-NoEncryption] [-SkipServerCertificateCheck | -SkipServerCNCheck] [-UseUDP] [-ClientCertificateThumbprint <String>] [-OutputSeverity {Default | Verbose}] [-Remove] 
+```
+
+### Cmdlet parameters for `Set-AzSSyslogForwarder`
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:127]: Content revised. Thx.

File: azure-stack/hci/concepts/other-security-features.md [hunk 4]
```diff
@@ -0,0 +1,518 @@
...
+
+The following parameters are for the `Set-AzSSyslogForwarder` cmdlet
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:129]: Content revised. Thx.

File: azure-stack/hci/concepts/other-security-features.md [hunk 5]
```diff
@@ -0,0 +1,518 @@
...
+|ServerPort |Port number the syslog server is listening on. |UInt16 |Yes |
+|NoEncryption |Force the client to send syslog messages in clear text. |Flag |No |
+|SkipServerCertificateCheck |Skip validation of the certificate provided by the syslog server during initial TLS handshake. |Flag |No |
+|SkipServerCNCheck |Skip validation of the Common Name value of the certificate provided by the syslog server during initial TLS handshake. |Flag |No |
+|UseUDP |Use syslog with UDP as transport protocol. |Flag |No |
+|ClientCertificateThumbprint |Thumbprint of the client certificate used to communicate with syslog server. |String |No |
+|OutputSeverity |Level of output logging. Values are Default or Verbose. Default includes severity levels: warning, critical, or error. Verbose includes all severity levels: verbose, informational, warning, critical, or error. |String |No |
+|Remove |Remove current syslog forwarder configuration and stop syslog forwarder. |Flag |No |
+
+### Configure the syslog forwarder with TCP, mutual authentication, and TLS 1.2 encryption
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:143]: Content revised. Thx.

File: azure-stack/hci/concepts/other-security-features.md [hunk 6]
```diff
@@ -0,0 +1,518 @@
...
+   ```azurepowershell
+   Set-AzSSyslogForwarder -ServerName <FQDN or ip address of syslog server> -ServerPort <Port number on which the syslog server is listening on> -ClientCertificateThumbprint <Thumbprint of the client certificate>
+   ```
+
+> [!IMPORTANT]
+> The client certificate must contain a private key. If the client certificate is signed using a self-signed root certificate, you must import the root certificate as well.
+
+Here is an example to set and import certificates for mutual authentication.
+
+This example script must be run from DVM.
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:165]: Content revised. Thx.

File: azure-stack/hci/concepts/other-security-features.md [hunk 7]
```diff
@@ -0,0 +1,518 @@
...
+
+In this example, certificates should have been stored as pfx files on DVM.
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:167]: Content revised throughout. Thx.

File: azure-stack/hci/concepts/other-security-features.md [hunk 8]
```diff
@@ -0,0 +1,518 @@
...
+   } 
+
+   $actionPlanInstanceId = Invoke-Command @params -ScriptBlock { 
+  
+   param($ServerName, $ServerPort, $ClientCertThumbprint) 
+   return Set-AzSSyslogForwarder -ServerName $ServerName -ServerPort $ServerPort -ClientCertificateThumbprint $ClientCertThumbprint 
+   }
+   ```
+
+## Configuring syslog forwarder with TCP, Server authentication, and TLS 1.2 encryption
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:281]: Content revised. Thx.

File: azure-stack/hci/concepts/other-security-features.md [hunk 9]
```diff
@@ -0,0 +1,518 @@
...
+
+   ```azurepowershell
+   Set-AzSSyslogForwarder -ServerName <FQDN or ip address of syslog server> -ServerPort <Port number on which the syslog server is listening on>  
+   -SkipServerCertificateCheck
+   ```
+
+> [!IMPORTANT]
+> Microsoft recommends against the use of the `-SkipServerCertificateCheck` flag in production environments.
+
+## Configuring syslog forwarder with TCP and no encryption
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:310]: Content revised. Thx.

File: azure-stack/hci/concepts/other-security-features.md [hunk 10]
```diff
@@ -0,0 +1,518 @@
...
+In this configuration, the syslog client in Azure Stack HCI forwards messages to the syslog server over TCP, with no encryption. The client doesn’t verify the identity of the server nor does it provide its own identity to the server for verification.
+
+```azurepowershell
+Set-AzSSyslogForwarder -ServerName <FQDN or ip address of syslog server> -ServerPort <Port number on which the syslog server is listening on> -NoEncryption
+```
+
+> [!IMPORTANT]
+> Microsoft recommends against using this configuration in production environments.
+
+## Configuring the syslog forwarder with UDP and no encryption
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:321]: Content revised. Thx.

File: azure-stack/hci/concepts/other-security-features.md [hunk 11]
```diff
@@ -0,0 +1,518 @@
...
+```azurepowershell
+Set-AzSSyslogForwarder -ServerName <FQDN or ip address of syslog server> -ServerPort <Port number on which the syslog server is listening on> -UseUDP
+```
+
+While UDP with no encryption is the easiest to configure, it doesn’t provide any protection against man-in-the-middle attacks or eavesdropping of messages.
+
+> [!IMPORTANT]
+> Microsoft recommends against using this configuration in production environments.
+
+## Removing the syslog forwarder configuration
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:334]: Content revised. Thx.

File: azure-stack/hci/concepts/other-security-features.md [hunk 12]
```diff
@@ -0,0 +1,518 @@
...
+
+Run the following command to remove the syslog forwarder configuration and stop the syslog forwarder:
+
+```azurepowershell
+Set-AzSSyslogForwarder -Remove 
+```
+
+## Verifying the syslog setup
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:342]: Content revised. Thx.

File: azure-stack/hci/concepts/other-security-features.md [hunk 13]
```diff
@@ -0,0 +1,518 @@
...
+
+Cmdlet parameters for the `Get-AzSSyslogForwarder` cmdlet:
+
+|Parameter |Description |Type |Required |
+|----|----|----|----|
+|Local |Show currently used configuration on current host. |Flag |No |
+|PerNode |Show currently used configuration on each host. |Flag |No |
+|Cluster |Show currently global configuration on Azure Stack HCI. This is the default behavior if no parameter is provided. |Flag |No |
+
+## Enabling and disabling the syslog forwarder
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:370]: Content revised. Thx.

Removed H2, promoted H3s to H2s.

File: azure-stack/hci/concepts/other-security-features.md [hunk 14]
```diff
@@ -0,0 +1,518 @@
...
+
+### Enable the syslog forwarder
+
+```azurepowershell
+Enable-AzSSyslogForwarder [-Force]
+```
+
+Syslog forwarder will be enabled with the stored configuration provided by last successful `Set-AzSSyslogForwarder` call. The cmdlet will fail if no configuration has been provided using `Set-AzSSyslogForwarder`.
+
+### Disable the syslog forwarder**
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:382]: Content revised. Thx.

File: azure-stack/hci/concepts/other-security-features.md [hunk 15]
```diff
@@ -0,0 +1,518 @@
...
+
+Parameter for `Enable-AzSSyslogForwarder` and `Disable-AzSSyslogForwarder` cmdlets:
+
+|Parameter |Description |Type |Required |
+|----|----|----|----|
+|Force |If specified, an action plan will always be triggered even if the target state is the same as current. This can be helpful when out-of-band changes need to be reset. |Flag |No |
+
+## Syslog message schema
+
+The syslog forwarder of the Azure Stack HCI infrastructure sends messages formatted following the BSD syslog protocol defined in RFC3164. Common Event Format (CEF) is also used to format the syslog message payload.
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:396]: Content revised. Thx.

File: azure-stack/hci/concepts/other-security-features.md [hunk 16]
```diff
@@ -0,0 +1,518 @@
...
+|MasProviderEventSourceName |  |
+|MasProviderGuid | AEA1B4FA-97D1-45F2-A64C-4D69FFFD92C9 |
+|MasProviderName |Microsoft-Windows-GroupPolicy |
+|MasSecurityUserId |Windows SID |
+|MasTask |0 |
+|MasTaskCategory |Process Creation |
+|MasUserData | KB4093112!!5112!!Installed!!0x0!!WindowsUpdateAgent Xpath: /Event/UserData/* |
+|MasVersion |0 |
+
+### Miscellaneous events
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:462]: Content revised. Thx.

File: azure-stack/hci/concepts/other-security-features.md [hunk 17]
```diff
@@ -0,0 +1,518 @@
...
+CN=MetricServiceAgent         10/2/2023 12:00:00 AM 10/1/2025 12:00:00 AM CN=AzureStackCertificationAuthority      4096 RSA-PKCS1-KeyEx      
+
+CN=PMCService                 10/2/2023 12:00:00 AM 10/1/2025 12:00:00 AM CN=AzureStackCertificationAuthority      4096 RSA-PKCS1-KeyEx      
+
+CN=ECEngineAgentService       10/2/2023 12:00:00 AM 10/1/2025 12:00:00 AM CN=AzureStackCertificationAuthority      4096 RSA-PKCS1-KeyEx      
+
+CN=URP                        10/2/2023 12:00:00 AM 10/1/2025 12:00:00 AM CN=AzureStackCertificationAuthority      4096 RSA-PKCS1-KeyEx   
+```
+
+This new component enables three major capabilities:
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:70]: Content revised. Thx.

File: azure-stack/hci/concepts/other-security-features.md [hunk 18]
```diff
@@ -0,0 +1,518 @@
...
+Run the following cmdlet to manually start a certificate rotation in your Azure Stack HCI cluster:
+
+> [!Note:]
+> This action will take place using LCM actions and will last about 10 minutes, depending on the size of the cluster.  This action affects all nodes that use LCM, and requires the user to belong to the `LCM authorization` group (PREFIX-ECESG) and `CredSSP` for remote Powershell invocation or remote desktop protocol (RDP) connection.
+
+```Powershell
+Start-SecretRotation 
+```
+
+## SIEM Syslog forwarder agent
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:85]: Content revised. Thx.
Updated H2 heading as suggested.

File: azure-stack/hci/concepts/other-security-features.md [hunk 19]
```diff
@@ -0,0 +1,518 @@
...
+
+For customers and organizations that require their own local SIEM, Azure Stack HCI 23H2 comes with an integrated mechanism that enables customers to forward security-related events to a local SIEM.
+
+Azure Stack HCI has an integrated syslog forwarder that, once configured, emits syslog messages defined in RFC3164, with the payload in Common Event Format (CEF).
+
+The following diagram illustrates integration of Azure Stack HCI with an external SIEM. All audits, security logs, and alerts are collected on each host and exposed via syslog with the CEF payload.  
+
+:::image type="content" source="media/other-security-features/integration-of-azure-stack-hci-with-external-siem.png" alt-text="The following diagram describes the integration of Azure Stack HCI with an external SIEM." border="false" lightbox="media/other-security-features/integration-of-azure-stack-hci-with-external-siem.png":::
+
+### Syslog forwarder agent capabilities
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:97]: Content revised. Thx.

File: azure-stack/hci/concepts/other-security-features.md [hunk 20]
```diff
@@ -0,0 +1,518 @@
...
+  >[!IMPORTANT]
+  > To protect against man-in-the-middle attacks and eavesdropping of messages, Microsoft strongly recommends that you use TCP with authentication and encryption in production environments.
+
+## Configuring SYSLOG Security Events forwarder
+
+Configuring syslog forwarder requires access to the physical host using a domain administrator account. A set of PowerShell cmdlets have been added to all Azure Stack HCI hosts to control behavior of the syslog forwarder.
+
+The `Set-AzSSyslogForwarder` cmdlet is used to set the syslog forwarder configuration for all hosts. If successful, an action plan instance will be started to config the syslog forwarder agents across all hosts. The action plan instance ID will be returned.
+
+### PowerShell cmdlet reference
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:119]: Content revised. Thx.

File: azure-stack/hci/concepts/other-security-features.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:129]: Content revised. Thx.

File: azure-stack/hci/concepts/other-security-features.md [hunk 21]
```diff
@@ -0,0 +1,518 @@
...
+    Remove-Item -Path $certPath 
+   } 
+  
+   Remove-PSSession -Session $session 
+   } 
+ 
+   Import self-signed root certificate
+   ```
+
+   Use this section only under applicable case.
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:228]: Content revised. Thx.

File: azure-stack/hci/concepts/other-security-features.md [hunk 22]
```diff
@@ -0,0 +1,518 @@
...
+   Import-PfxCertificate -FilePath $certPath -CertStoreLocation "Cert:\\LocalMachine\root" -Password $certPassword 
+   Remove-Item -Path $certPath 
+   } 
+   Remove-PSSession -Session $session 
+   }
+   ```
+
+3. Set the syslog forwarder to use mutual authentication
+
+   An action plan will be started and its ID will be stored in `$actionPlanInstanceId`. Please monitor the action plan and validate that it completes successfully.
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:265]: Content revised. Thx.

File: azure-stack/hci/concepts/other-security-features.md [hunk 23]
```diff
@@ -0,0 +1,518 @@
...
+ms.date: 10/27/2023
+---
+
+# Other security features for Azure Stack HCI (preview)
+
+[!INCLUDE [applies-to](../../includes/hci-applies-to-supplemental-package.md)]
+
+This article describes new components in the security layer of the Azure Stack HCI solution for security information and event management (SIEM).
+
+## Renamed local built-in users
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:18]: Content revised. Thx.

File: azure-stack/hci/concepts/other-security-features.md [hunk 24]
```diff
@@ -0,0 +1,518 @@
...
+
+|Name |Enabled |Description |
+|-----|-----|-----|
+|ASBuiltInAdmin |True |Built-in account for administering the computer/domain |
+|ASBuiltInGuest |False |Built-in account for guest access to the computer/domain |
+
+> [!Important]
+> We recommend that you create your own local administrator account, and also that you disable the well-known `RID 500` user account.
+
+The `Guest` account is protected by the security baseline drift control mechanism. For more information, see [Security baseline settings for Azure Stack HCI (preview)](/azure-stack/hci/concepts/secure-baseline).
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:30]: Pinging the SME > Carlos.

File: azure-stack/hci/concepts/other-security-features.md [hunk 25]
```diff
@@ -0,0 +1,518 @@
...
+We also added two net new components to Azure Stack HCI in 23H2 release, as follows:
+
+- Secrets management engine.
+- SIEM Syslog forwarder agent.
+
+## Secrets management engine
+
+The new Lifecycle Manager (LCM) stack requires multiple new components to maintain secure communications with other infrastructure resources and services. To ensure security, we implemented internal secret creation and rotation capabilities.
+
+All new services exposed internally by LCM have authentication or encryption certificates associated with them. When you look into your HCI nodes you will see several new certificates created under the path `LocalMachine/Personal certificate store (Cert:\LocalMachine\My).
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:41]: Content revised. Thx.

File: azure-stack/hci/concepts/other-security-features.md [hunk 26]
```diff
@@ -0,0 +1,518 @@
...
+
+# Other security features for Azure Stack HCI (preview)
+
+[!INCLUDE [applies-to](../../includes/hci-applies-to-supplemental-package.md)]
+
+This article describes new components in the security layer of the Azure Stack HCI solution for security information and event management (SIEM).
+
+## Renamed local built-in users
+
+We renamed the local built-in users for the `RID 500 local administrator` and `RID 501 guest` accounts. The new names and status of those accounts are as follows:
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:20]: Content revised. Thx.

File: azure-stack/hci/concepts/other-security-features.md (hunk 24 above)
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:30]: Carlos signs off on the suggestion. 
Content revised. Thx.

File: azure-stack/hci/concepts/other-security-features.md [hunk 27]
```diff
@@ -0,0 +1,518 @@
...
+The `Guest` account is protected by the security baseline drift control mechanism. For more information, see [Security baseline settings for Azure Stack HCI (preview)](/azure-stack/hci/concepts/secure-baseline).
+
+We also added two net new components to Azure Stack HCI in 23H2 release, as follows:
+
+- Secrets management engine.
+- SIEM Syslog forwarder agent.
+
+## Secrets management engine
+
+The new Lifecycle Manager (LCM) stack requires multiple new components to maintain secure communications with other infrastructure resources and services. To ensure security, we implemented internal secret creation and rotation capabilities.
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:39]: Content revised. Thx.

File: azure-stack/hci/concepts/other-security-features.md [hunk 28]
```diff
@@ -0,0 +1,518 @@
...
+
+```Powershell
+Start-SecretRotation 
+```
+
+## SIEM Syslog forwarder agent
+
+Security compliance requires strict log and audit of security events; in Azure Stack HCI, we recommend that customers use our Azure Cloud Sentinel service. For more information, see [Microsoft Sentinel](https://azure.microsoft.com/products/microsoft-sentinel).
+
+For customers and organizations that require their own local SIEM, Azure Stack HCI 23H2 comes with an integrated mechanism that enables customers to forward security-related events to a local SIEM.
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:89]: Content revised. Thx.

File: azure-stack/hci/concepts/other-security-features.md [hunk 29]
```diff
@@ -0,0 +1,518 @@
+---
+title: Other security features for Azure Stack HCI (preview)
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:2]: Content revised. Thx.

File: azure-stack/hci/concepts/other-security-features.md [hunk 30]
```diff
@@ -0,0 +1,518 @@
...
+
+### Syslog forwarder agent capabilities
+
+Syslog forwarder agents are deployed on every Azure Stack HCI host. Each of the agents will forward syslog messages collected from the host they are on, to the customer configured syslog server.
+
+Syslog forwarder agents work independently from each other but can be managed all together on any one of the hosts. You can use provided cmdlets with administrative privileges on any host to control the behavior of all forwarder agents.
+
+The syslog forwarder in Azure Stack HCI supports the following configurations:
+
+- **Syslog over TCP with mutual authentication (client and server) and TLS 1.2 encryption:** In this configuration, both the syslog server and the syslog client can verify the identity of each other via certificates. Messages are sent over a TLS 1.2 encrypted channel.
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:105]: Content revised to add jumps. Thx.

File: azure-stack/hci/concepts/other-security-features.md (hunk 29 above)
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/concepts/other-security-features.md:2]: To address your comment about linking to related security articles, I added at Line 18:

For additional security considerations, see:

- [Security baseline settings on Azure Stack HCI (preview)](/azure-stack/hci/concepts/secure-baseline).
- [BitLocker encryption on Azure Stack HCI (preview)](/azure-stack/hci/concepts/security-bitlocker).

If additional links should be added, please advise. Thx.

**Pull request merged**